alpine-server/src/main/java/alpine/server/auth/JsonWebToken.java

```diff
@@ -130,13 +130,30 @@ public String createToken(final Principal principal, final List<Permission> perm
      * @return a String representation of the generated token
      * @since 1.8.0
      */
-    public String createToken(final Principal principal, final List<Permission> permissions, final IdentityProvider identityProvider) {
+    public String createToken(final Principal principal, final List<Permission> permissions,
+            final IdentityProvider identityProvider) {
+        final int ttl = Config.getInstance().getPropertyAsInt(Config.AlpineKey.AUTH_JWT_TTL);
+        return createToken(principal, permissions, identityProvider, ttl);
+    }
+
+    /**
+     * Creates a new JWT for the specified principal. Token is signed using
+     * the SecretKey with an HMAC 256 algorithm.
+     *
+     * @param principal the Principal to create the token for
+     * @param permissions the effective list of permissions for the principal
+     * @param identityProvider the identity provider the principal was authenticated with. If null, it will be derived from principal
+     * @param ttlSeconds the token time-to-live in seconds
+     * @return a String representation of the generated token
+     * @since 2.2.5
+     */
+    public String createToken(final Principal principal, final List<Permission> permissions, final IdentityProvider identityProvider, final int ttlSeconds) {
         final Date today = new Date();
```

Comment:
Consider renaming `today` to `now` so it matches the `addDays` -> `addSeconds` change.